src/Riok.Mapperly/Descriptors/Enumerables/CollectionInfoBuilder.cs

```diff
@@ -52,42 +52,46 @@ private record CollectionTypeInfo(
         new CollectionTypeInfo(CollectionType.ImmutableSortedSet, typeof(ImmutableSortedSet<>), Immutable: true),
         new CollectionTypeInfo(CollectionType.ImmutableQueue, typeof(ImmutableQueue<>), Immutable: true),
         new CollectionTypeInfo(CollectionType.IImmutableQueue, typeof(IImmutableQueue<>), Immutable: true),
-        new CollectionTypeInfo(CollectionType.IImmutableQueue, typeof(IImmutableQueue<>), Immutable: true),
         new CollectionTypeInfo(CollectionType.ImmutableStack, typeof(ImmutableStack<>), Immutable: true),
         new CollectionTypeInfo(CollectionType.IImmutableStack, typeof(IImmutableStack<>), Immutable: true),
         new CollectionTypeInfo(CollectionType.IImmutableDictionary, typeof(IImmutableDictionary<,>), Immutable: true),
         new CollectionTypeInfo(CollectionType.ImmutableDictionary, typeof(ImmutableDictionary<,>), Immutable: true),
+        new CollectionTypeInfo(CollectionType.ImmutableSortedDictionary, typeof(ImmutableSortedDictionary<,>), Immutable: true),
         new CollectionTypeInfo(CollectionType.Span, typeof(Span<>)),
         new CollectionTypeInfo(CollectionType.ReadOnlySpan, typeof(ReadOnlySpan<>)),
         new CollectionTypeInfo(CollectionType.Memory, typeof(Memory<>)),
         new CollectionTypeInfo(CollectionType.ReadOnlyMemory, typeof(ReadOnlyMemory<>)),
     };
 
-    public static CollectionInfos? Build(WellKnownTypes wellKnownTypes, ITypeSymbol source, ITypeSymbol target)
+    public static CollectionInfos? BuildCollectionInfos(WellKnownTypes wellKnownTypes, ITypeSymbol source, ITypeSymbol target)
```

Comment:
Is there a benefit in the new longer name? Isn't this part already included in the class name?

Reply:
`private static CollectionInfo Build(WellKnownTypes wellKnownTypes, ITypeSymbol type, ITypeSymbol enumeratedType)` has the same signature as `public static CollectionInfos? Build(WellKnownTypes wellKnownTypes, ITypeSymbol source, ITypeSymbol target)`.

I've renamed the private one.

Reply:
Ah got it.